.meteor/versions

```diff
@@ -46,6 +46,8 @@ es5-shim@4.6.14_1
 factory@1.0.0
 fastclick@1.0.12
 force-ssl@1.0.12
+fortawesome:fontawesome@4.4.0_1
+fourseven:scss@3.8.1
```

Comment:
I believe these are added by Bert (https://github.com/themeteorchef/bert/blob/master/package.js#L17-L18). Not sure if it's safe to remove them if we don't use them. I could also look for another notification system with no dependencies (should be an easy drop-in).